serles/backends/certbot.py

```diff
@@ -0,0 +1,68 @@
+import subprocess
+import tempfile
+import json
+
+from subprocess import STDOUT, PIPE
+from pathlib import Path
+
+class Backend:
+    """ Serles Backend for certbot
+
+    Backend will pass the certificate signing request to an already configured certbot installation.
+
+    Example use cases:
+
+       1. Want actual trusted certs without using an internal CA (you want it to work in browsers/etc. out of the box)
+       2. Don't want the services externally exposed (internal services/development only)
+       3. Don't want the internal services having control over your external DNS (random developers/users)
+       4. Have an intermediate server running serles that you are willing to grant external DNS update rights to.
+
+    This allows any internal entity to transparently use serles ACME CA with http-01 validation, but the actual signing
+    requests are delegated to external ACME CA.
+
+    """
+
+    def __init__(self, config):
+        self.path = "/usr/bin/certbot"
```

Comment:
```suggestion
        self.path = "certbot"
```

should take it from the `$PATH`, right?

Reply:
yeah, i know (this is one of the possibly bad ideas i mentioned above). given that serles is (hopefully) started through an init daemon, it's path should be well defined. i'm ok with leaving it at `/usr/bin`, but maybe add a check that the file exists and is executable

Reply:
Updated, have a look at current code.